zuul.d/jobs.yml

```diff
@@ -0,0 +1,9 @@
+---
+- job:
+    name: ansible-test-sanity
+    description: ansible-test sanity
+    required-projects:
+      - name: ansible/ansible
+        override-checkout: stable-2.5
+    vars:
+      python_version: 2.7 
```

Comment:
maybe use: python_version: python2.7

Reply:
In this case I want just the version so I can do
`ansible-test [sanity,network-integration] --python 2.7 ...`